### ISSUE TITLE
have build container only claim port 1313 if running `make serve`

### ISSUE THREAD

**Issue by craigbox** [istio/istio#51299]
You can only run the build container once, because it port-forwards port 1313.  
```
Makefile.overrides.mk:CONTAINER_OPTIONS = -p 1313:1313 ${ADDITIONAL_CONTAINER_OPTIONS}
```

This is only necessary if you're using it to run a local Hugo preview, and it stops you being able to do useful things like `make gen` or `make lint-md` while it's running.

Please move this option only to the target for `make serve`.

**Affected product area (please put an X in all that apply)**

[x] Docs
[x] Test and Release

**Comment by zirain** [comment]
is this for istio.io repo?

**Comment by craigbox** [comment]
Yes, sorry, I thought it was one of the common files.  I'll move the issue to the right repo!
